database/index.js

```diff
@@ -18,8 +18,10 @@ let imageSchema = mongoose.Schema({
 //compile the schema into a Model
 let Images = mongoose.model('Images', imageSchema);
 
+let addOne = (imagesData) => Images.create(imagesData);
```

Comment:
Is there a reason to have an add function here? I thought the detail Images is only pulling from the Image service so not sure when this would be used

Reply:
^ Ignore - looks like you're using it for your seeder

Reply:
@this is for the seed.js